Update mainnet and testnet nodes to v1.3.2, devnet to chain ID 48

clay-aptos opened the pull request:
### Description

### Test Plan
<!-- Please provide us with clear details for verifying that your changes work. -->

gregnazario commented on developer-docs-site/docs/releases/devnet-release.md:
```diff
@@ -11,7 +11,7 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
 ## Aptos Framework
 
-Aptos Framework Release commit hash - `1d0dc2c75da1e397effc8bcfa14114652462654d`
+Aptos Framework Release commit hash - `framework-mainnet`
```
Need to confirm this, framework hasn't been updated yet

clay-aptos replied:
Ack. Converting to Draft until we know. Thanks!

clay-aptos replied:
Per the team, we can remove this section and value for the Devnet network.

clay-aptos commented:
Looks like we should rev the CLI too:
https://github.com/aptos-labs/aptos-core/pull/7465

clay-aptos commented:
> Looks like we should rev the CLI too: #7465

@gregnazario I updated the CLI on testnet to aptos-cli-v1.0.9. Please let me know if I should do the same in mainnet and devnet. Thanks!

clay-aptos commented:
> > Looks like we should rev the CLI too: #7465
> 
> @gregnazario I updated the CLI on testnet to aptos-cli-v1.0.9. Please let me know if I should do the same in mainnet and devnet. Thanks!

Nevermind. I forgot we use the same CLI version across networks. Fixed!
